doc/02.-Prerequisites.md

```diff
@@ -60,8 +77,22 @@ Reboot then to allow the machine to do its policy refresh.
 
 To log in as a user of the domain, press the link **"Not listed?"** in the greeter. Then enter the username followed by the password.
 
-By default, there is no default domain configured in SSSD. You have to enter the full user name with one of the forms `USER@DOMAIN` or `DOMAIN/USER`.
+### SSSD
+
+By default, there is no default domain configured in SSSD. You have to enter the full user name with one of the forms `USER@DOMAIN.COM` or `DOMAIN/USER`.
 
 On the first log in the user's home directory is created.
 
 All of this (default domain, default path for home directories, default shell, etc.) is configurable in `/etc/sssd/sssd.conf`.
+
+### Winbind
+
+If Winbind is used as a backend, the account has to be specified in the following format: `DOMAIN\\USER`.
```

Comment:
hum, winbind doesn’t support USER@DOMAIN too? I don’t remember with the amount of testing we did, you are probably right, but just checking :)

Reply:
\o/